Styling/README.md

```diff
@@ -501,6 +502,18 @@ Returns the cosine value of the number (in radians).
 }
 ```
 
+#### sqrt 
+
+`sqrt(value : Number) : Number`
```

Comment:
What is the range of `value`?  `>= 0.0`?  What is the behavior if, for example, `-1.0` is provided?